st2tests/st2tests/fixtures/packs/test_content_version_fixture/fixture.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The StackStorm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from st2tests import fixturesloader
+
+PACK_NAME = "test_content_version"
+_, PACK_PATH = fixturesloader.get_fixture_name_and_path(__file__)
+PACK_PATH = PACK_PATH[: -len("_fixture")]
```

Comment:
The `test_content_version` fixture is in a git submodule. I wanted to keep all the fixture details in this repo so that they're available even if someone has not cloned the submodule. So, I put the fixture vars in `test_content_version_fixture` directory and adjusted the vars.